modules/apps/powwow/powwow-portlet/docroot/meetings/calendar_booking_recurrence_container.jspf

```diff
@@ -309,7 +297,11 @@ if (recurrence != null) {
 								label: '<liferay-ui:message key="done" />',
```

Comment:
should we think about enable/disable the button `Done` when the violation happens?

Reply:
It is difficult to enable/disable button of liferay window because we can't identify id of name of button